bin/node/runtime/src/lib.rs

```diff
@@ -1701,7 +1701,7 @@ impl frame_benchmarking_pallet_pov::Config for Runtime {
 }
 
 construct_runtime!(
-	pub enum Runtime where
+	pub struct Runtime where
```

Comment:
Did you change the `construct_runtime` macro itself?  
Otherwise this obviously wont compile.

Reply:
Already done: https://github.com/paritytech/substrate/pull/11932

Reply:
At last academy 😜

Reply:
Ah okay. The failing UI tests were from a change to their files, I see.